src/main/java/uk/gov/hmcts/reform/civil/advice/ControllerExceptionHandler.java

```diff
@@ -39,4 +40,10 @@ public ResponseEntity<Object> documentUploadException(DocumentUploadException do
         log.error(documentUploadException.getMessage());
         return new ResponseEntity<>("Document upload unsuccessful", new HttpHeaders(), HttpStatus.BAD_REQUEST);
     }
+
+    @ExceptionHandler(PartyIdsUpdatedException.class)
+    public ResponseEntity<Object> partyIdsUpdatedException(PartyIdsUpdatedException partyIdsUpdatedException) {
+        log.error(partyIdsUpdatedException.getMessage());
```

Comment:
@Gareth40342 this wont log out the full object - in case we accidentally log names / case data on prod